Source/WebCore/workers/WorkerFontLoadRequest.cpp

```diff
@@ -84,10 +84,13 @@ bool WorkerFontLoadRequest::ensureCustomFontData()
             contiguousData = m_data.takeAsContiguous();
         convertWOFFToSfntIfNecessary(contiguousData);
         if (contiguousData) {
-            m_fontCustomPlatformData = FontCustomPlatformData::create(*contiguousData, m_url.fragmentIdentifier().toString());
-            m_data = WTFMove(contiguousData);
-            if (!m_fontCustomPlatformData)
+            auto fontCustomPlatformData = FontCustomPlatformData::create(*contiguousData, m_url.fragmentIdentifier().toString());
+            if (!fontCustomPlatformData) {
                 m_errorOccurred = true;
+                return false;
+            }
+            lazyInitialize(m_fontCustomPlatformData, fontCustomPlatformData.releaseNonNull());
+            m_data = WTFMove(contiguousData);
```

Comment:
This looks like a behavior change since `m_data` no longer gets set if `FontCustomPlatformData::create()` returns null. Is that OK?